### ISSUE TITLE
Add Login App Details to Login Page

### ISSUE THREAD

**Issue by shervine** [askmench/mench-web-app#2456]
TODO

- [x] Add product descriptions for Login Apps https://mench.com/entities/7372
- [x] Add to Platform cache
- [x] Show on the /login page so users know what they will access when login

**Comment by shervine** [comment]
@grumo Check this out: https://mench.com/login

**Comment by grumo** [comment]
@shervine it's a good idea to inform users but it can be confusing since most users will only have a specific use case when they login. For example, a company doesn't need to see the miner, user, and chatbot definitions. It would make more sense to have either a generic login page or separate login pages for each use case.

**Comment by shervine** [comment]
@grumo I was looking at this as an educational stand-point. At this stage neither companies nor users will login as users will mostly use Messenger webview and companies don't have their app ready yet. But when they do, my goal was to educate them by letting them know what products we have for each side of our marketplace and how we can help them. Plus how can they get confused if they see their own use case in the list?

**Comment by grumo** [comment]
@shervine  I think the problem is with the line "Login to your Mench account to access:" this implies that all users will have access to all the apps which is not true. It would make more sense to say "Depending on your account type you may have access to one or more of the following items:". But this again is too wordy and confusing. Maybe saying something more generic that applies to all user types would make more sense.

**Comment by shervine** [comment]
@grumo Yes I agree, we can discuss more tomorrow
